lib/bootleg/tasks/phoenix_digest.ex

```diff
@@ -1,13 +1,14 @@
 defmodule Bootleg.Tasks.PhoenixDigest do
-  @moduledoc "Installs needed depedencies and calls `mix phoenix.digest`."
-  alias Bootleg.UI
+  @moduledoc "Installs dependencies and calls `mix phoenix.digest`."
+  alias Bootleg.{Config, UI}
 
   use Bootleg.Task do
     task :phoenix_digest do
+      mix_env = Config.get_config(:mix_env, "prod")
```

Comment:
It would be better to use the documented DSL verb `config`, and not an undocumented internal function here.